snapcraft/internal/lifecycle.py

```diff
@@ -88,6 +97,15 @@ def init():
     with open(snapcraft_yaml_path, mode='w') as f:
         f.write(yaml)
 
+    git_cache = os.path.join('.git')
+    if os.path.exists(git_cache):
+        if os.path.exists('.gitignore'):
+            pass
+        else:
```

Comment:
Rather than passing, perhaps this would be easier to read:

```python3
# Don't mess with an existing .gitignore
if not os.path.exists('.gitignore'):
    with open('.gitignore', mode='w') as f:
        f.write(_TEMPLATE_GITIGNORE)
```